redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/databaseserver/responses/SslConfigResponse.java

```diff
@@ -0,0 +1,46 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.responses;
+
+import java.util.Set;
+
+import com.sequenceiq.redbeams.api.endpoint.v4.databaseserver.requests.SslMode;
+import com.sequenceiq.redbeams.doc.ModelDescriptions;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(description = ModelDescriptions.SSL_CONFIG_RESPONSE)
+public class SslConfigResponse {
```

Comment:
I suggest renaming this to `SslConfigV4Response` to be consistent with other request/response classes.